FlowCrypt/Functionality/Mail Provider/Message Provider/ProcessedMessage.swift

```diff
@@ -36,10 +36,12 @@ struct ProcessedMessage {
             case .unsigned:
                 return "message_not_signed".localized
             case let .error(message):
-                return "message_signature_verify_error".localizeWithArguments(message.lowercasingFirstLetter())
+                let signaturErrorString = "message_signature_verify_error".localized
```

Comment:
`signaturErrorString` -> `signatureErrorString`

Reply:
Oops. thanks for notice